cli/cmd/inject.go

```diff
@@ -128,14 +138,14 @@ func injectObjectMeta(t *metaV1.ObjectMeta, k8sLabels map[string]string, options
  */
 func injectPodSpec(t *v1.PodSpec, identity k8s.TLSIdentity, controlPlaneDNSNameOverride string, options *injectOptions, report *injectReport) bool {
 	report.hostNetwork = t.HostNetwork
-	report.sidecar = healthcheck.HasExistingSidecars(t)
+	report.sidecar = healthcheck.HasExistingSidecars(t, true)
```

Comment:
curious about setting `onlyThirdParty` to `true` here... wouldn't we not want to proceed with injection if a `linkerd-proxy` was found?

Reply:
Great catch! It actually doesn't matter what that boolean is because at this point the uninjection should have already occurred so `linkerd-proxy` should not be present. So it actually would make sense to use `false` here as a sanity safeguard. That means we're not using `false` anywhere so I'm gonna revert the addition of that argument :stuck_out_tongue: